apps/laboratory/tests/shared/utils/email.ts

```diff
@@ -84,4 +84,8 @@ export class Email {
 
     return `web3modal${index}@mailsac.com`
   }
+
+  getSmartAccountEnabledEmail(): string {
+    return 'web3modal-smart-account@mailsac.com'
```

Comment:
Do we keep the list of available MailSac accounts somewhere? How do we create & manage them?

Reply:
this is a deprecated method. mailsac account addresses can be found at 
`/validators/ModalWalletValidator.ts`

will remove this method and move the list to this utils file